perf/benchmark/runner/fortio.py

```diff
@@ -139,8 +139,10 @@ def sync_fortio(url, table, selector=None, promUrl="", csv=None, csv_output="",
 
     data = []
     for filename in os.listdir(temp_dir_path):
+        print(filename)
         with open(os.path.join(temp_dir_path, filename), 'r') as f:
-            data_dict = json.load(f)
+            print(f.read())
+            data_dict = json.load(f, strict=False)
```

Comment:
1. can you instead catch `json.decoder.JSONDecodeError` and log the contents of the file here?
2. after f.read(), you can you check you are not getting and EOF since file pointer would have moved.

If you fix these two, lgtm.

Reply:
Yes, sure!